### PR TITLE
createpvd and savepvd functions

### PR DESCRIPTION
**Pull request by oriolcg**
support for `.pvd` files (collection of `.pvtu` files). The user can simply use the same API as the in the serial case:
```julia
  createpvd(parts,"filename") do pvd
    for (x,t) in xt
      pvd[t] = createvtk(Ω,"filename_$t",cellfields=["x"=>x])
    end
  end
```

### CODE REVIEW

**Comment by oriolcg** [comment]
This PR is linked to the `Gridap` PR gridap/Gridap.jl#736

File: src/Visualization.jl [hunk 1]
```diff
@@ -152,3 +152,36 @@ function Visualization.create_vtk_file(
       celldata=c,nodaldata=n)
   end
 end
+
+struct DistributedPvd{T}
+  pvds::AbstractPData{T}
```
Comment (fverdugo) [inline src/Visualization.jl:157]: This struct is not type-stable. Simply define it like this:

```julia
struct DistributedPvd{T<:AbstractPData}
  pvds::T
end
```

**Review by fverdugo — CHANGES_REQUESTED** [review]

File: src/Visualization.jl (hunk 1 above)
Comment (oriolcg) [reply src/Visualization.jl]: fixed in https://github.com/oriolcg/GridapDistributed.jl/commit/a0b6f85e434d51ced94c61b1b2033c6960aee742

**Comment by oriolcg** [comment]
Hi @amartinhuertas, @fverdugo. After the release of `Gridap@0.17.8` this PR passes the checks and can be merged.

**Comment by amartinhuertas** [comment]
> Hi @amartinhuertas, @fverdugo. After the release of Gridap@0.17.8 this PR passes the checks and can be merged.

Many thanks for your work @oriolcg ! 

Can you please add an entry to the NEWS.md file with the feature implemented in this PR?

**Review by fverdugo — APPROVED** [review]

**Comment by oriolcg** [comment]
> Can you please add an entry to the NEWS.md file with the feature implemented in this PR?

Done in 4efec6fd46c29f40f808ba005ca4b52a9e9f78a5

**Pull request merged**